backend/migrations/versions/98e8c06e83ba_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 98e8c06e83ba
+Revises: 7408c872d6c0
+Create Date: 2019-07-31 18:36:12.649210
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '98e8c06e83ba'
+down_revision = '7408c872d6c0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('participant', sa.Column('first_name', sa.String(length=50), nullable=True))
```

Comment:
We should use `alter_column` here, not `drop_column` and `add_column` as altering will preserve data.

Reply:
It has been done automatically by alembic

Reply:
Not sure about this one with alembic. 😕

Reply:
@magul, we are waiting for your re-review

Reply:
I understand, that Alembic could create such migration, but that's just a tool and in that case, it does it wrong. We're not on production right now, but as a learning exercise, we should prepare that migration in such manner, that it could be safely run on a production environment.

Right now that migration will drop existing columns an create new ones. Data previously stored in removed columns are permanently lost. I belive we should use here https://alembic.sqlalchemy.org/en/latest/ops.html?highlight=unique#alembic.operations.Operations.alter_column

Reply:
@magul 
I made all the requested changes in migration files. I checked the command flask db upgrade and flask db migrate - everything worked. When I opened the database in dbeaver it had column names: first_name and last_name. I hope that now everything works ok, but it would be nice if you could check :)